packages/components/psammead-brand/src/index.jsx

```diff
@@ -45,6 +46,11 @@ const StyledWrapper = styled.div`
   }
 `;
 
+const ConstraintWrapper = styled.div`
+  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
```

Comment:
Has this been tested with some of the larger brand logos? Would like to see how the Russian rand looks on a smaller device with this new max width.

However this might be more of a UX problem to sort first.

Reply:
Agreed we will need to talk to ux about brand handling, however this max width will only affect devices over 1280px wide so we should be good :)

Reply:
Great. Thanks for clarifying.